public/New-DbaDbUser.ps1

```diff
@@ -28,9 +28,12 @@ function New-DbaDbUser {
     .PARAMETER Login
         When specified, the user will be associated to this SQL login and have the same name as the Login.
 
-    .PARAMETER Username
+    .PARAMETER UserName
```

Comment:
Please revert, this is improper spelling/casing of the word.

Reply:
Thanks for spotting this, It's slipped through

Reply:
It was mentioned in the fields section.  https://learn.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.management.smo.usertype?view=sql-smo-160#fields

`SqlLogin	0	Specifies a SQLLogin user. This is deprecated, use SQLUser instead.`

I might be wrong. I thought the smo library is part of `dbatools` or `dbatools.library` module as a dependency which should be the recent version. If this is the case then I will prefer to keep the changes. However, if the module uses smo library installed locally then it makes sense to revert it back. I was not clear, your inputs will be helpful.

Reply:
Thanks for spotting, it slipped through.